### ISSUE TITLE
Isometry of hermitian spaces

### ISSUE THREAD

**Issue by StevellM** [thofma/Hecke.jl#619]
With the following setup
```
Qx, x = QQ["x"]
K, a = number_field(x^3+3, "a")
Kt, t = K["t"]
E, b = number_field(t^2-2, "b")

V = hermitian_space(E, E[102 b; -b 0])
H = hermitian_space(E, E[0 1; 1 0])
  
infp = infinite_places(K)
```
I am encountering this issue: 
```
julia> isisometric(V,H)
true
julia> isisometric(V,H, infp[1])
false
```
So my spaces are globally isometric but not everywhere locally... I am not sure about where this is coming from. The spaces are both regular and indefinite. Any idea on which test is not working correctly here ?

**Comment by StevellM** [comment]
So after further investigation I may have found the problem: `isisometric` is not testing isometry at the real places. So the function should check equality of the rank, of the determinants norm classes and that the signatures at the real places are the same. Is that right ?

**Comment by thofma** [comment]
Yes, that sounds correct. The spaces need to be regular for that (according to Kirschmer's Habilitation).
